src/main/scala/com/microsoft/hyperspace/actions/RefreshActionBase.scala

```diff
@@ -99,17 +99,10 @@ private[actions] abstract class RefreshActionBase(
    * Finally, append the previously known deleted files to the result. These
    * are the files for which the index was never updated in the past.
    */
-  protected lazy val deletedFiles: Seq[String] = {
+  protected lazy val deletedFiles: Seq[FileInfo] = {
     val relation = previousIndexLogEntry.relations.head
     val originalFiles = relation.data.properties.content.fileInfos
-    val delFiles = (originalFiles -- currentFiles).map(_.name)
```

Comment:
Question: Why is the part for deduplication removed from deletedFiles and appendedFiles functions? (They have been added recently thru #214).

Reply:
Yes, seems we still need to remove duplicates. To my mind, it's a bit of mess to handle duplicates here but #198 will resolve the problem; with "since" semantic, we don't need to care about appendedFiles and deletedFiles of previousIndexLogEntry.